Mathlib/Algebra/Group/Subgroup/Even.lean

```diff
@@ -0,0 +1,85 @@
+/-
+Copyright (c) 2024 Artie Khovanov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Artie Khovanov
+-/
+import Mathlib.Algebra.Group.Even
+import Mathlib.Algebra.Group.Subgroup.Defs
+
+/-!
+# Squares and even elements
+
+This file defines the subgroup of squares / even elements in an abelian group.
+-/
+
+namespace Subsemigroup
+variable {S : Type*} [CommSemigroup S] {a : S}
+
+variable (S) in
+/--
+In a commutative semigroup `S`, `Subsemigroup.sqIn S` is the subsemigroup of squares in `S`.
+-/
+@[to_additive evenIn
```

Comment:
Could you please add a line to the `to_additive` file that teaches it to translate `squareIn` to `evenIn`?